typeclasses/baseobject.py

```diff
@@ -0,0 +1,19 @@
+from evennia import DefaultObject
```

Comment:
Ok I did all the cuts, but I'm stuck here. I don't know how to set these properties on objects.Object, let all the typeclasses inherit from there, but without doing multiple inheritance. If I change the parent class, they won't get any of the defaults from the evennia code I guess, and super can be used only on direct parent class, right?
sorry mate I'm too noobie to solve this. For now I just doubled the code, adding properties into Room, Character and Item typeclass